src/Native/System.Native/pal_networking.cpp

```diff
@@ -1272,6 +1272,21 @@ static int32_t GetIPv6PacketInformation(cmsghdr* controlMessage, IPPacketInforma
     return 1;
 }
 
+cmsghdr* GET_CMSG_NXTHDR(msghdr* mhdr, cmsghdr* cmsg)
+{
+#ifndef _GLIBC_
```

Comment:
There was an `_` missing like `__GLIBC__` I have just added it; which should only be defined for non-musl libc standard headers such as on all Ubuntus which ship with glibc, OS X and BSDs.